frappe/integrations/offsite_backup_utils.py

```diff
@@ -110,7 +110,7 @@ def generate_files_backup():
 
 	backup = BackupGenerator(
 		frappe.conf.db_name,
-		frappe.conf.db_name,
+		frappe.conf.db_user or frappe.conf.db_name,
```

Comment:
I think it's best to use `frappe.conf.db_user` everywhere and make sure it's always set instead of defining these fallbacks everywhere (like `frappe.conf.db_user ||= frappe.conf.db_name`).

Reply:
All existing site configs will have to be updated for this to work. :eyes:

Reply:
We can write a patch but we can't do patch + removing fallback together AFAIK. Maybe we should write a patch for now and remove fallbacks from next version.

Reply:
I meant during `frappe.init`, not in the site config